release/202008/metrics-value-rc.md

```diff
@@ -8,17 +8,42 @@ _The value working group is undergoing a restructuring. All previously released
 **Goal:**  
 Understand if the project valuable to its community of users (including downstream projects) or contributors?  
 
+<div>
+<table>
+  <thead><tr><th>Metric/Details</th><th>Question</th><th>Provide Feedback</th></tr></thead>
+<tbody>
+  <tr><td><a href="https://chaoss.community/metric-project-velocity/">Project Velocity</a></td><td>What is the development speed for an organization?</td></tr>
+  <tr><td><a href="https://github.com/chaoss/wg-value/blob/master/focus-areas/communal-value/social-currency-metric-system.md">Social Currency Metric System (SCMS) [Under Review]</a></td><td>How does one measure the value of community interactions and accurately gauge “trust” within a community as evident from qualitative sentiment?</td><td><a href="https://github.com/chaoss/wg-value/issues/75">Issue #75</a></td></tr>
+</tbody>
+</table>
+</div>
+
+
 ### Focus Area - Individual Value
 
 **Goal:**  
 Understand if the project is valuable to an individual user or contributor?
 
+<div>
+<table>
+  <thead><tr><th>Metric/Details</th><th>Question</th><th>Provide Feedback</th></tr></thead>
```

Comment:
The rest of the table is missing this column. Not needed here (or do we have this column on all tables?
```suggestion
  <thead><tr><th>Metric/Details</th><th>Question</th></tr></thead>
```

Reply:
We need it in all columns. I will edit it.